### PR TITLE
docs: add how to proxy dubbo in apisix blog

### PR DESCRIPTION
**Pull request by lijing-21**
add how to proxy dubbo in apisix blog

### CODE REVIEW

**Review by moonming — APPROVED** [review]

**Comment by moonming** [comment]
@wangchengming666 please take a look, thx

File: content/zh/blog/news/how-to-proxy-dubbo-in-apache-apisix.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+     * @param context pass http infos
+     * @return Map<String, Object></> pass to response http
+     **/
+    Map<String, Object> apisixDubbo(Map<String, Object> httpRequestContext);
+}
+```
+
+如上所示，Dubbo 接口的定义是固定的。即方法参数中 `Map` 表示 APISIX 传递给 Dubbo Provider 关于 HTTP request 的一些信息（如：header、body...)。而方法返回值的 `Map` 表示 Dubbo Provider 传递给 APISIX 要如何返回 HTTP response 的一些信息。
+
+接口信息之后可通过 XML 配置方式发布 DemoService。
```
Comment (wangchengming666) [inline content/zh/blog/news/how-to-proxy-dubbo-in-apache-apisix.md:47]: @lijing-21 @moonming 感觉这句话有点不通顺，是不是缺少谓语？或者你想表达的是：接口信息配置好之后？

Comment (lijing-21) [reply content/zh/blog/news/how-to-proxy-dubbo-in-apache-apisix.md]: Yes, thanks for the review. I have fixed that~

**Review by wangchengming666 — APPROVED** [review]
LGTM

**Comment by lijing-21** [comment]
@wangchengming666 hi, sorry to bother you again. 

Could you please help me merge this PR? I don't know who else to ask for help. Thanks a lot.

**Comment by moonming** [comment]
@AlbumenJ please take a look, thx

**Pull request merged**